extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js

```diff
@@ -54,6 +57,27 @@ define('wikia.articleVideo.featuredVideo.adsConfiguration', [
 		return {};
 	}
 
+	function loadMoatTrackingPlugin() {
+		if (!win.moatjw) {
+			var scriptElement = win.document.createElement('script');
+			scriptElement.async = true;
+			scriptElement.src = moatJwplayerPluginUrl;
+			win.document.head.appendChild(scriptElement);
+		}
+	}
+
+	/**
+	 * Calculate depth
+	 *
+	 * @param {number} depth
+	 * @returns {number} rv
+	 */
+	function calculateRV(depth) {
```

Comment:
Can we reuse method from `articleVideoAd` module?

Reply:
Refactored.